src/Schema/ScreenOwnerType.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * All valid screen (page) types in Eigen
+ * @packageDocumentation
+ */
+export enum ScreenOwnerType {
+  Artist = "Artist",
```

Comment:
We are using `lowerCamelCase` for all naming conventions except for enum/interface names.